src/inspectorscripts.ts

```diff
@@ -135,6 +136,14 @@ def _jupyterlab_variableinspector_is_matrix(x):
     return False
 
 
+def _jupyterlab_variableinspector_is_widget(x):
+    try:
+        from ipywidgets import DOMWidget
```

Comment:
Can we move this import into the `_check_imported()` method. Then instead of using try-except for control flow here, we can just use `if 'ipywidgets' in sys.modules: ...`

Reply:
Yes that makes sense, let's follow the same pattern.

Reply:
Moved to `_check_imported`, but keeping the `try except` to stay consistent with the other imports.

Reply:
Also wondering whether these imports should be prefixed in some way, to avoid conflicting or leaking to the notebook / user? (but that's another discussion).